frontend/src/views/activity/SideBarProgram.vue

```diff
@@ -42,6 +49,14 @@ export default {
       return this.addDays(this.startOfDay, 1)
     }
   },
+  watch: {
+    apiScheduleEntries (value) {
+      this.scheduleEntries = value.items.map(entry => defineHelpers(entry, true))
+    }
+  },
+  beforeMount () {
+    this.scheduleEntries = this.apiScheduleEntries.items.map(entry => defineHelpers(entry, true))
+  },
```

Comment:
Possibly we'd want to share this with `CampProgram`.

Reply:
Also, instead of duplicating the logic in watcher and beforeMount, use the `immediate` flag on the watcher:
```js
watch: {
  apiScheduleEntries: {
    immediate: true,
    handler: function (value) {
      this.scheduleEntries = value.items.map(entry => defineHelpers(entry, true))
    }
  }
}
```
See https://vuejs.org/v2/api/#vm-watch